lua/autorun/server/sv_acf_legal_checks.lua

```diff
@@ -1,8 +1,14 @@
 local legalChecks = {
     acf_ammo = function( ent )
+        local parent = ent:GetParent()
+
+        if IsValid( parent ) and ( parent:GetClass() == "gmod_wire_hologram" or parent:GetClass() == "starfall_hologram" ) then
```

Comment:
I'd leave it as a suggestion, but GH won't let me - just make a lookup table for the parent class and simplify this line. No reason to get lazy here :p